### DISCUSSION TITLE
Gibt es überhaupt noch eine Community Edition?

### DISCUSSION THREAD

**Discussion by 2greatsecret** [tine20/tine20#7494] [Q&A] [answered]
Nach "erfolgreichem" Update auf Ellie 2023.11.1, bekomme ich in der obersten Zeile (links von Benutzernamen) angezeigt, dass meine Lizenz abgelaufen ist. Was habe ich falsch gemacht?

**Comment by pschuele** [comment, marked as answer]
Hallo 2greatsecret,
Die Ellie 2023.11 hat tatsächlich keine separate Community Edition mehr (-> https://github.com/tine-groupware/tine).
In diesem Repo wird es aber noch einige Zeit die Pakete aus der 2022er-Reihe geben.
Dazu wird es demnächst auch noch eine Ankündigung geben.

**Reply by pschuele** [reply]
hier findet ihr das community jahresrelease für 2023 -> https://github.com/tine20/tine20/releases/tag/2023.12.1
